src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -617,9 +624,9 @@ type Update
           if b.Equals currentModel' newModel' then
             []
           else
-            this.Recursive((ValueSome currentModel'), getCurrentModel >> b.Get, newModel', b.Dispatch, b.Binding)
+            this.Recursive((ValueSome currentModel'), getCurrentModel >> b.Get, newModel', b.Binding)
       | AlterMsgStream b ->
-          this.Recursive(currentModel |> ValueOption.map b.Get, getCurrentModel >> b.Get, b.Get newModel, b.Dispatch, b.Binding)
+          this.Recursive(currentModel |> ValueOption.map b.Get, getCurrentModel >> b.Get, b.Get newModel, b.Binding)
```

Comment:
I was thinking that this change is a bug.  But it is not because of what [you said in your comment](https://github.com/elmish/Elmish.WPF/pull/489#issuecomment-1244098621): that the altered `dispatch` function is baked in during initialization.

Reply:
The altered `dispatch` doesn't need to be passed down here because it was already passed down during binding initialization:

https://github.com/elmish/Elmish.WPF/blob/fed258a7a97c95df3927b8d6a7bfaaf82b2dd4d1/src/Elmish.WPF/BindingVmHelpers.fs#L436-L441

Reply:
Now I think there was no bug before this PR and no bug after.

Furthermore, does `AlterMsgStreamBinding` need a `Dispatch` field any more?  I think not.  Can you double check that?

Reply:
I don't think it does, and I think the same is true for `LazyBinding.Dispatch`

Reply:
Done and it seems to work.